index.d.ts

```diff
@@ -199,6 +199,12 @@ export interface IFieldGroupWidgetSettings {
   [setting: string]: WidgetSettingsValue
 }
 
+export interface IEntryEditor {
+  widgetNamespace: 'editor-builtin' | 'builtin' | 'extension' | 'app',
+  widgetId: string,
+  settings?: IEditorInterfaceOptions
+}
+
```

Comment:
nit: can we doc block these so we know what builtin, editor-builtin, etc mean? Not a blocker, just nice to have.